test/e2e/e2e_suite_test.go

```diff
@@ -82,7 +83,8 @@ func TestE2E(t *testing.T) {
 	if prowArtifactFolder, exists := os.LookupEnv("ARTIFACTS"); exists {
 		artifactFolder = prowArtifactFolder
 	}
-
+	SetDefaultEventuallyTimeout(2 * time.Minute)
```

Comment:
IMO we should not have SetDefaultEventuallyTimeout & SetDefaultEventuallyPollingInterval
Relying on those values makes it harder to detect which interval variables are not defined, and ultimately reduces the portability of tests to other infrastructure providers.

Reply:
If we don't set these, default timeout will be used which is 1sec. We may want to skip providing different intervals for each scenarios, that is why I think adding a reasonable timeout makes sense. WDYT?